### PR TITLE
feature/OB-16 added layer

### PR DESCRIPTION
**Pull request by Alinochka07**
task is done

### CODE REVIEW

**Review by MillanaCh — DISMISSED** [review]

File: public/images/4_1.png
Comment (Alex-2kZharkov) [inline public/images/4_1.png:1]: Almost good. let's give a more meaningful name to this image. Something like `hero-overlay`

File: src/app/page.tsx [hunk 1]
```diff
@@ -1,17 +1,27 @@
...
+        after:h-32
+        after:bg-repeat-x
+        after:bg-[top_0.125rem_center]
+        after:bg-[length:50rem]
+        after:left-0
+        after:w-full"
+      >
         <Header />
+        <PromoBanner barbershopCount={barbsershopData.barbershopCount} />
+        <div className="bg-[url('/images/4_1.png')] w-full h-full bg-[center_top] scale-75 bg-cover z-10 absolute" />
```
Comment (Alex-2kZharkov) [inline src/app/page.tsx]: Don't forget to fix name here

File: src/app/page.tsx [hunk 2]
```diff
@@ -1,17 +1,27 @@
...
+    <main className="overflow-hidden flex md:container md:max-w-screen-2xl mx-auto flex-col bg-header-bg">
+      <div
+        className="flex flex-col justify-between bg-[url('/images/bg/hero.jpg')] bg-no-repeat bg-[left_-3.4rem_bottom_0.4rem]
+        relative
+        bg-cover
+        after:bg-[url('/images/bg/white-grunge.svg')]
+        after:z-10
+        after:h-32
+        after:bg-repeat-x
+        after:bg-[top_0.125rem_center]
```
Comment (Alex-2kZharkov) [inline src/app/page.tsx]: I would change that to `....5rem_center` because the main page will look better

**Review by Alex-2kZharkov — CHANGES_REQUESTED** [review]

File: public/images/4_1.png
Comment (Alinochka07) [reply public/images/4_1.png:1]: done

File: src/app/page.tsx (hunk 1 above)
Comment (Alinochka07) [reply src/app/page.tsx]: yes

File: src/app/page.tsx (hunk 2 above)
Comment (Alinochka07) [reply src/app/page.tsx]: changed to 5rem. Also fixed the extra space below the background image, bg can be shrunk now. The problem was occurring due to the transform:scale(). Now it looks much better.

**Comment by Alinochka07** [comment]
Fixed.

File: src/app/page.tsx (hunk 2 above)
Comment (Alex-2kZharkov) [reply src/app/page.tsx]: Excellent job! I am really proud of you🙌

**Review by Alex-2kZharkov — DISMISSED** [review]

**Review by MillanaCh — DISMISSED** [review]

**Review by Alex-2kZharkov — APPROVED** [review]

**Pull request merged**